src/promptflow/promptflow/executor/_tool_resolver.py

```diff
@@ -106,7 +107,12 @@ def _convert_node_literal_input_types(self, node: Node, tool: Tool, module: type
                 updated_inputs[k].value = create_image(v.value)
             elif isinstance(value_type, ValueType):
                 try:
-                    updated_inputs[k].value = value_type.parse(v.value)
+                    updated_inputs[k].value = v.value
+                    # Don't convert value type for literal node input.
```

Comment:
Then I think we don't need the following converting error?

Reply:
No, we still need to handle composite objects. Such as list of images.
But it only happen with specific image format.